pkg/blobstore/replication/remote_blob_replicator.go

```diff
@@ -12,22 +12,22 @@ import (
 	"google.golang.org/grpc"
 )
 
-type remoteBlobReplicator struct {
+type GrpcBlobReplicator struct {
 	source           blobstore.BlobAccess
 	replicatorClient replicator.ReplicatorClient
 }
 
-// NewRemoteBlobReplicator creates a BlobReplicator that forwards
+// NewGrpcBlobReplicator creates a BlobReplicator that forwards
 // requests to a remote gRPC service. This service may be used to
 // deduplicate and queue replication actions globally.
-func NewRemoteBlobReplicator(source blobstore.BlobAccess, client grpc.ClientConnInterface) BlobReplicator {
-	return &remoteBlobReplicator{
+func NewGrpcBlobReplicator(source blobstore.BlobAccess, client grpc.ClientConnInterface) BlobReplicator {
```

Comment:
GRPC should be capitalized, per the Go style guidelines, just like HTTPClient, etc.